From beaa5b64c60d501fa9cd59fdc563e5f5bccf6882 Mon Sep 17 00:00:00 2001
From: Qing He <qing.he@intel.com>
Date: Tue, 31 Aug 2010 22:51:58 +0800
Subject: [PATCH] valgrind: fix perl scripts

this is a temporary patch to workaround cross compilation.
otherwise @PERL@ will be replaced to perl-native binary,
this creates unusable scripts and fails FILERDEPENDS mechanism
(esp. rpm)

a better fix would need:
 1. configure.ac should differentiate PERL and HOSTPERL
 2. optionally remove ${STAGING_DIR} in #! line before do_install

8/31/2010 - created by Qing He <qing.he@intel.com>

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Maxin B. John <maxin.john@intel.com>

---
 massif/ms_print.in  | 2 +-
 perf/vg_perf.in     | 2 +-
 tests/vg_regtest.in | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/massif/ms_print.in b/massif/ms_print.in
index a206ce4..df1bc31 100755
--- a/massif/ms_print.in
+++ b/massif/ms_print.in
@@ -1,4 +1,4 @@
-#! @PERL@
+#! /usr/bin/perl
 
 ##--------------------------------------------------------------------##
 ##--- Massif's results printer                         ms_print.in ---##
diff --git a/perf/vg_perf.in b/perf/vg_perf.in
index 90ee1d2..c585096 100644
--- a/perf/vg_perf.in
+++ b/perf/vg_perf.in
@@ -1,4 +1,4 @@
-#! @PERL@
+#! /usr/bin/perl
 ##--------------------------------------------------------------------##
 ##--- Valgrind performance testing script                  vg_perf ---##
 ##--------------------------------------------------------------------##
diff --git a/tests/vg_regtest.in b/tests/vg_regtest.in
index 7152765..ad18800 100755
--- a/tests/vg_regtest.in
+++ b/tests/vg_regtest.in
@@ -1,4 +1,4 @@
-#! @PERL@
+#! /usr/bin/perl
 ##--------------------------------------------------------------------##
 ##--- Valgrind regression testing script                vg_regtest ---##
 ##--------------------------------------------------------------------##
